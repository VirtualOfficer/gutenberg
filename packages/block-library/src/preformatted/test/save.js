/**
 * External dependencies
 */
import renderer from 'react-test-renderer';

/**
 * Internal dependencies
 */
import Preformatted from '../save';

describe( 'core/more/save', () => {
	it( 'saves a simple content without crashing', () => {
		const wrapper = renderer.create( <Preformatted attributes={ { content: '<b>Hello World!</b>' } } /> );
		const rendered = wrapper.toJSON();
		expect( rendered ).toMatchSnapshot();
		wrapper.unmount();
	} );
} );
