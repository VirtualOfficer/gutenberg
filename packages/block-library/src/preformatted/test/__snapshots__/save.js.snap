// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`core/more/save saves a simple content without crashing 1`] = `
<pre>
  <div
    dangerouslySetInnerHTML={
      Object {
        "__html": "<b>Hello World!</b>",
      }
    }
  />
</pre>
`;
